q2_diversity/plugin_setup.py

```diff
@@ -147,6 +147,39 @@
         citations['faith1992conservation']]
 )
 
+plugin.methods.register_function(
+    function=q2_diversity.alpha_phylogenetic_alt,
+    inputs={'table': FeatureTable[Frequency],
+            'phylogeny': Phylogeny[Rooted]},
+    parameters={'metric': Str % Choices(alpha.phylogenetic_metrics())},
+    outputs=[('alpha_diversity',
+              SampleData[AlphaDiversity] % Properties('phylogenetic'))],
+    input_descriptions={
+        'table': ('The feature table containing the samples for which alpha '
+                  'diversity should be computed.'),
+        'phylogeny': ('Phylogenetic tree containing tip identifiers that '
+                      'correspond to the feature identifiers in the table. '
+                      'This tree can contain tip ids that are not present in '
+                      'the table, but all feature ids in the table must be '
+                      'present in this tree.')
+    },
+    parameter_descriptions={
+        'metric': 'The alpha diversity metric to be computed.'
+    },
+    output_descriptions={
+        'alpha_diversity': 'Vector containing per-sample alpha diversities.'
+    },
+    name='Alpha diversity (phylogenetic)',
```

Comment:
Can you change this to be `Alpha diversity (phylogenetic) - alternative implementation`. or `fast implementation` or something along those lines.

Reply:
yup!